cornedbeef/src/main/java/com/swiftkey/cornedbeef/PunchHoleCoachMark.java

```diff
@@ -137,6 +221,28 @@ public PunchHoleCoachMarkBuilder setOnGlobalClickListener(View.OnClickListener l
             return this;
         }
 
+        /**
+         * Set the color of this coach mark.
+         *
+         * @param overlayColor the color to set
+         */
+        public PunchHoleCoachMarkBuilder setOverlayColor(int overlayColor) {
+            this.overlayColor = overlayColor;
+            return this;
+        }
+
+        /**
+         * Set whether the punch hole animates going over the target view from
+         * left to right.
+         *
+         * @param horizontalTranslation
+         */
+        @TargetApi(Build.VERSION_CODES.HONEYCOMB)
```

Comment:
Add a comment to say that this won't work before honeycomb and probably and if statement to ensure that we never set it to true if that is the case